src/core/qgismobileapp.cpp

```diff
@@ -734,9 +733,16 @@ void QgisMobileapp::readProjectFile()
     title = mProject->title().isEmpty() ? mProjectFileName : mProject->title();
   }
 
+  QList<QPair<QString, QString>> projects = recentProjects();
```

Comment:
Here you could use a QMap< "path", "title" >, so the path can only be unique?

Reply:
We can't otherwise we'd lose ordering of the list (which is needed to insure most recent sits on top)